apx/common/src/apx_nodeManager.c

```diff
@@ -543,9 +543,13 @@ static void apx_nodeManager_createNode(apx_nodeManager_t *self, const uint8_t *d
                         {
                            APX_LOG_ERROR("[APX_NODE_MANAGER] createOutInitDataFromDefinition failed");
                         }
-                        apx_nodeData_setNodeInfo(nodeData, nodeInfo);
+			apx_nodeData_setNodeInfo(outDataFile->nodeData, nodeInfo);

```

Comment:
You're using tabs on this line only. Please change to spaces.

Reply:
Argh - did a quick vi edit - sorry